docs/db/week-3/lesson.md

```diff
@@ -101,21 +105,21 @@ Here the endpoint includes `:id`, which identifies an extra parameter in the URL
 Express provides a simple way to get such parameters from the request:
 
 ```js
-var custId = parseInt(req.params.id);
+const custId = parseInt(req.params.id);
 ```
 
 Note that the `id` in `req.params.id` must match the name after the colon in the endpoint `"customers/:id"`.
 
 Next we need to query the `customers` table and provide the value of the id into a WHERE clause so that we retrieve only the one row that matches:
 
 ```js
-db.query(
-  "SELECT * FROM customers WHERE id = $1",
-  [custId],
-  function (err, result) {
-    // TODO - more code here...
-  }
-);
+db.query("SELECT * FROM customers WHERE id = $1", [custId])
+  .then(function (result) {
+    console.log(result);
+  })
+  .catch(function (err) {
+    console.log(err);
+  });
```

Comment:
Shall we use arrow function here?

Reply:
Hey @jackhclee 

Could you take a look through https://github.com/CodeYourFuture/Module-Databases when you have a sec?

I'm debating letting L9 use the new coursework as well as the new DB module, but I really need some eyes on it first.

Reply:
@SallyMcGrath I agree re LDN9 moving to Github projects for the database module too
I had a cursory look at the Database Module and those issues made sense to me

Reply:
I'm going to clone it down now and check it makes sense

Reply:
I think PD might have a problem with this. It's unclear. We might find out tonight.

Reply:
Daniel has reviewed, but I forgot to ask Karen about L9 (and in fact this conversation).

Reply:
Why could there be a problem for PD with this, @SallyMcGrath ?

Reply:
Yes, this conversation has me and Mitch discussing L9 moving to the new coursework for the DB module. We have a completely new DB module for L9 and it has no coursework set for it. We have written new coursework, but it's in the [Module repo](https://github.com/CodeYourFuture/Module-Databases). It's ready to go, but this conversation is me saying that PD might have a problem with us using it, in which case we will have to use the old coursework.